### PR TITLE
Use bleach to clean HTML in code feedback.

### LINKED ISSUE #267: More accurate categorization of error type in code question?
As the python code is lazily evaluated, sometimes a problematic code from user will pass ``exec``, and the error won't be raised until it is runned in ``test_code``. That bring me tons of email. I know we can define some try-exception to prevent that, and use some code to generate the error messages, but we need to put that code in every ``test_code`` which might encounter the problem. Is it possible to create some ``Exception`` types, when which is raised, the runpy will categorize the exception to ``user_error`` instead of ``test_error`` so that won't trigger the error email?

### PR DESCRIPTION
**Pull request by dzhuang**
Fix #267 .

### CODE REVIEW

File: course/page/code.py [hunk 1]
```diff
@@ -770,13 +770,21 @@ def transfer_attr(name):
             raise RuntimeError("invalid runpy result: %s" % response.result)
 
         if hasattr(response, "feedback") and response.feedback:
+            def sanitize(s):
+                import bleach
+                return (
+                    bleach.clean(s, tags=["p", "pre"])
+                        # html5lib will turn <user code> to <user code="">
```
Comment (inducer) [inline course/page/code.py]: We could just switch to different delimiters there, e.g. `[user code]`.

File: course/page/code_feedback.py [hunk 2]
```diff
@@ -166,7 +166,7 @@ def call_user(self, f, *args, **kwargs):
             from traceback import format_exc
             self.add_feedback(
                     "<p>"
-                    "The function '%s' supplied in your code failed with "
+                    "The '%s' supplied in your code failed with "
```
Comment (inducer) [inline course/page/code_feedback.py]: Maybe replace "function" with "callable"? I'm not sure no noun is better.

File: course/page/code.py (hunk 1 above)
Comment (dzhuang) [inline course/page/code.py]: Thanks. Updated.

**Comment by inducer** [comment]
LGTM. Thanks for the contribution. I'll do a container rebuild sometime soon.

**Pull request merged**